templates/web/gloucester/footer_extra_js.html

```diff
@@ -0,0 +1 @@
+[% PROCESS 'footer_extra_js_base.html' highways=0 roadworks=0 validation=0 cobrand_js=0 %]
```

Comment:
Just noting as mentioned in Slack, I think three of these should probably be on?